### ISSUE TITLE
Accessibility: "Our Team" button tooltips accessibility

### ISSUE THREAD

**Issue by aduth** [GSA/centers-of-excellence#359]
The tooltips shown when focusing or hovering images on the ["Our Team" page](https://coe.gsa.gov/about/team.html) suffer from a few accessibility issues:

- Tooltip content shown on focus should be dismissible.
- Tooltip content shown on focus should persist to allow the user to hover its content.
- Tooltip content should be announced to assistive technology.

**WCAG Success Criteria (AA):** [1.4.13: Content on Hover or Focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)

**Relevant technique:** [Client Side Script SCR39: Making content on focus or hover hoverable, dismissible, and persistent](https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR39)

>ensure that authors who cause additional content to appear and disappear [in coordination with keyboard focus or pointer hover] must design the interaction in such a way that users can: [...] dismiss it without disrupting their page experience.

>Additional content that is displayed when a user moves the pointer over a trigger or moves the keyboard focus to the trigger (for example, a pop-up) must remain visible to allow users time to read and interact with the content and must allow the user to move the pointer over the additional content.

**Relevant failure:** [F95: Failure of Success Criterion 1.4.13 due to content shown on hover not being hoverable](https://www.w3.org/WAI/WCAG21/Techniques/failures/F95)

Additionally, I was not successful in having the tooltip text be announced using a combination of macOS Mojave (10.14.6), Safari (13.1.1), and VoiceOver. The JavaScript library in use ([Tippy.js](https://atomiks.github.io/tippyjs/)) _does_ appear to support announcements via screen reader, and the demos appear to work correctly in my testing. It is unclear to me if this is an issue of (a) using an out-of date version of the library, (b) the fact that the tooltip content is lengthy, or (c) the tooltip content contains complex markup.

**Remediation guidance:**

It may be worth reconsidering whether a tooltip is the ideal method to communicate this information, or if instead it could be displayed statically, optionally in combination with some activating event (clicking upon the image).
